e2e/connection_test.go

```diff
@@ -21,74 +21,114 @@ import (
 	"testing"
 
 	"github.com/cihub/seelog"
+	tequilapi_client "github.com/mysteriumnetwork/node/tequilapi/client"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestClientConnectsToNode(t *testing.T) {
+var (
```

Comment:
Isn't these values passed as e2e test args?

Reply:
Cant reuse at the moment